src/main/java/core/basesyntax/model/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 100;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = "WHITE";
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public Figure getRandomFigure(String shape) {
```

Comment:
That logic provides not random object. You define what `shape` you want to receive and provide that object.
Add some random here:)